src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,88 @@
+const { sendErrorResponse } = require('../helpers/sendErrorMessage');
+const {
+  expensesService,
+  initExpensesService,
+} = require('../services/expenses.service');
+const { validateExpenses } = require('../validators/expenses.validator');
+const { validateId } = require('../validators/users.validator');
+const { STATUS_CODES, ERRORS } = require('../variables/variables');
+
+const expensesUrl = '/expenses';
+
+const expensesController = (server) => {
+  initExpensesService();
+
+  server.get(expensesUrl, (req, res) => {
```

Comment:
Create routes and use your controllers in them, for example
```javascript
const expensesRoutes = express.Router();

expensesRoutes.get('/', expensesController.getAll);
// other routes
```
Then you can use routers:
```javascript
app.use('/users', usersRouter);
app.use('/expenses', expensesRouter);
```